### ISSUE TITLE
Config files for the experiments from blog post

### ISSUE THREAD

**Issue by elephantmipt** [HazyResearch/zoology#5]
Thank you for the blog posts and detailed wandb reports on the results! 
Our group is thrilled to reproduce the results on synthetic tasks from part two. It would be helpful if you could provide the configuration files for the experiments, so that we can be absolutely sure we are using the same hyperparameters and architecture.

**Comment by simran-arora** [comment]
Hi, thank you for your interest!

You can produce the synthetic figures from the blogs using this experiment launch code. Instructions for how to launch a sweep using one of these experiment files are provided in the README. The sweep covers a range of hyperparameters (learning rates) for each sequence mixer and we reported the best accuracy for each mixer in each setting across the learning rates. Other parameters should not need updating.
https://github.com/HazyResearch/zoology/blob/main/zoology/experiments/mqar_d_model.py

You can toggle this list in the above file to pick the mixers to evaluate: https://github.com/HazyResearch/zoology/blob/23f6bef6f9057f74dc10c8e19abb26ba4554481a/zoology/experiments/mqar_d_model.py#L132

You can then construct plots as appear in the blogs using this analysis code. Note you will need to update for the relevant sweep ids in your code to pull from wandb appropriately if you choose to use this analysis code. 
https://github.com/HazyResearch/zoology/blob/23f6bef6f9057f74dc10c8e19abb26ba4554481a/zoology/analysis/mqar_d_model.py#L58

Please let us know if you have additional questions.

**Comment by simran-arora** [comment]
Closing. Please reopen if you have additional questions.
